app/index.html

```diff
@@ -32,55 +33,63 @@
-        <div class="l-welcome-footer-logo">
-          <a href="https://quickblox.com" target="_blank"><img src="images/logo-quickblox.svg" alt="QuickBlox"></a>
+        <footer class="l-welcome-footer l-flexbox l-flexbox_column l-flexbox_flexbetween is-hidden">
+          <button class="btn l-welcome_btn l-welcome_btn_green j-twitterDigits">connect with phone number</button>
+          <button id="loginQB" class="btn l-welcome_btn l-welcome_btn_white j-login_QB">login by email or social</button>
+        </footer>
+        <div class="l-spinner">
+          <div class="spinner">
+            <div class="spinner-dot1"></div>
+            <div class="spinner-dot2"></div>
+          </div>
         </div>
-      </footer>
-      <div class="l-spinner">
-        <div class="spinner">
-          <div class="spinner-dot1"></div>
-          <div class="spinner-dot2"></div>
+        <div class="l-welcome_links">
+          <div class="l-welcome_footer_logo">
+            <a href="https://quickblox.com" target="_blank">
+              <div class="i-mini_logo"></div>
+              <span class="by_QB">by QuickBlox</span>
+            </a>
+          </div>
+          <div class="l-welcome_download">
+            <a class="l-welcome_download_app" href="https://itunes.apple.com/us/app/q-municate/id909698517?ls=1&mt=8" target="_blank">
+              <div class="i-appstore"></div>
+            </a>
+            <a class="l-welcome_download_app" href="https://play.google.com/store/apps/details?id=com.quickblox.q_municate" target="_blank">
+              <div class="i-android"></div>
+            </a>
+          </div>
         </div>
       </div>
     </section>
 
     <section id="loginPage" class="l-page is-hidden">
-      <div class="l-container l-flexbox l-flexbox_column l-flexbox_flexbetween">
-        <header class="l-welcome-header l-flexbox l-flexbox_column l-flexbox_flexbetween">
-          <img class="l-welcome-header-logo" src="images/logo-qmunicate-big.svg" alt="Q-municate">
-          <img class="l-welcome-header-text" src="images/logo-qmunicate-text.svg" alt="Q-municate">
-          <div class="download">
-            <a class="download-app" href="https://play.google.com/store/apps/details?id=com.quickblox.q_municate" target="_blank"><img src="images/app-android.svg" alt="download Q-municate Android"></a>
-            <a class="download-app" href="https://itunes.apple.com/us/app/q-municate/id909698517?ls=1&mt=8" target="_blank"><img src="images/app-ios.svg" alt="download Q-municate iOS"></a>
+      <div class="l-welcome_container">
+        <div class="l-welcome_block_login l-welcome_logo_login"></div>
+        <div class="l-welcome_block l-welcome_text_logo_login">q-municate</div>
+        <div class="l-welcome_block l-welcome_text_login">Log In</div>
+        <form class="l-form l-form_login" action="#">
+          <span class="text_error">Error</span>
+          <div class="input-field login_email">
```

Comment:
Uses another class name 'login_email' because input has the same class name. It's confused me